pkg/apis/application/v1alpha1/types_test.go

```diff
@@ -1130,9 +1130,9 @@ func TestAppProjectSpec_DestinationClusters(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			d := AppProjectSpec{Destinations: tt.destinations}
-			if got := d.DestinationClusters(); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("AppProjectSpec.DestinationClusters() = %v, want %v", got, tt.want)
-			}
+			got := d.DestinationClusters()
+			ok := reflect.DeepEqual(got, tt.want)
+			require.Truef(t, ok, "AppProjectSpec.DestinationClusters() = %v, want %v", got, tt.want)
```

Comment:
I think `assert.Equal` would actually work fine here. We're just comparing string slices.